takwimu/templates/takwimu/_includes/navbar/about.html

```diff
@@ -8,18 +8,79 @@
+                 target="_blank" rel="noopener">
+                <i class="fab fa-twitter fa-fw text-white"></i>
+              </a>
+            </li>
+            <li class="nav-item pt-3 pb-3">
+              <a class="nav-link border-right border-light" href="https://facebook.com/TakwimuAfrica"
+                 target="_blank" rel="noopener">
+                <i class="fab fa-facebook fa-fw text-white"></i>
+              </a>
+            </li>
+            <li class="nav-item pt-3 pb-3">
+              <a class="nav-link border-right border-light" href="https://medium.com/takwimu-africa"
+                 target="_blank" rel="noopener">
+                <i class="fab fa-instagram fa-fw"></i>
+              </a>
+            </li>
+            <li class="nav-item pt-3 pb-3">
+                <a class="nav-link border-right border-light" href="https://medium.com/takwimu-africa"
+                   target="_blank" rel="noopener">
+                  <i class="fab fa-slack fa-fw "></i>
+                </a>
+              </li> 
+            <li class="nav-item pt-3 pb-3">
+              <a class="nav-link" href="https://github.com/TakwimuAfrica"
+                 target="_blank" rel="noopener">
+                <i class="fab fa-github fa-fw"></i>
+              </a>
+            </li>
+          </ul>
+    </div>
+    </div>
+
+    <div class="row pt-5">
+      <div class="col-12">
+          <p class="mb-0 pl-5" style="opacity: 0.6; font-size: 16px">Takwimu is an initiative by:</p>
+          <div class="row pl-5">
+            <a href="https://www.gatesfoundation.org" target="_blank" rel="noopener"
+                class="col-2 partner-logo-container m-2 px-1 d-flex align-items-center justify-content-center">
+              <img src="{% static 'img/logos/bmgf-white.png' %}" class="img-responsive" alt="Bill and Melinda Gates Foundation">
+            </a>
+            <a href="http://www.africapractice.com/" target="_blank" rel="noopener"
+                class="col-2 partner-logo-container m-2 px-1 d-flex align-items-center justify-content-center">
+              <img src="{% static 'img/logos/ap-white.png' %}" class="img-responsive" alt="africapractice">
+            </a>
+            <a href="https://codeforafrica.org" target="_blank" rel="noopener"
+                class="col-2 partner-logo-container m-2 px-3 d-flex align-items-center justify-content-center">
+              <img src="{% static 'img/logos/cfafrica-white.png' %}" class="img-responsive" alt="Code for Africa">
+            </a>
+            <a href="https://ihub.co.ke/" target="_blank" rel="noopener"
+                class="col-2 partner-logo-container m-2 px-4 d-flex align-items-center justify-content-center">
+              <img src="{% static 'img/logos/ihub-white.png' %}" class="img-responsive" alt="iHub">
+            </a>
+          </div>
     </div>
 </div>
+<style>
+.border-light{
```

Comment:
This class fades the social media icons and makes them harder to see.

![184](https://user-images.githubusercontent.com/1779590/42723028-9e873390-875e-11e8-8873-f9d8695d2ee8.png)